lib/jekyll-seo-tag/drop.rb

```diff
@@ -103,6 +103,11 @@ def json_ld
         @json_ld ||= JSONLDDrop.new(self)
       end
 
+      # JSON-LD output, sorted by the key, as JSON.
+      def json_ld_sorted_json
+        JSON.parse(json_ld.to_json).sort_by(&:first).to_h.to_json
```

Comment:
Parsing a JSON string just to sort the Hash object seems sub-optimal to me.
How about sorting the hash at source directly?

Also, `json_ld_sorted_json` sounds repetitive.
How about simply `sorted_json_ld`?

```ruby
def sorted_json_ld
  json_ld.to_h.compact.sort_by(&:first).to_h.to_json
end
```

Reply:
Since the `to_json` method is doing a bunch of work, it's probably best if I just sort there.